kernels/streamer_alu/streamer_add_stream.preprocfinal.mlir

```diff
@@ -55,26 +50,5 @@ func.func public @streamer_add(%arg0 : memref<?xi64, "L3">, %arg1 : memref<?xi64
   func.return
 }
 
-"accfg.accelerator"() <{"name" = @snax_alu, 
-  "fields" = {
-    "loop_bound_0" = 960 : i32, 
-    "a_tstride_0" = 961 : i32, 
-    "b_tstride_0" = 962 : i32, 
-    "c_tstride_0" = 963 : i32, 
-    "a_sstride_0" = 964 : i32, 
-    "b_sstride_0" = 965 : i32, 
-    "c_sstride_0" = 966 : i32, 
-    "a_ptr" = 967 : i32, 
-    "b_ptr" = 968 : i32, 
-    "c_ptr" = 969 : i32, 
-    "alu_mode" = 972 : i32, 
-    "loop_bound_alu" = 973 : i32}, 
-  "launch_fields" = {
-    "launch_streamer" = 970 : i32, 
-    "launch_alu" = 974 : i32}, 
-  "barrier" = 975 : i32}> {
-    "streamer_config" = #snax.streamer_config<r[1, 1], r[1, 1], r[1, 1]>
-} : () -> ()
-
 func.func private @snax_is_compute_core() -> i1
```

Comment:
Same comment as in other test, maybe it makes sense to keep this in? Not sure what you think is easiest